src/PullCountStackChart.tsx

```diff
@@ -56,9 +57,11 @@ export class PullCountStackChart extends React.Component<IPullCountStackChartPro
 
   /**
    * Format label tooltips
+   * Required to return a emtpy string as the labels prop is required
+   * to display the custom tooltip.
    */
   formatLabel = (point: any): string => {
-    return `${point.datum.name}: ${point.datum.y} pulls`;
+    return '';
```

Comment:
You can probably remove this method and instead define an inline arrow function to `labels`. Ex:

```typescript
labels={() => ''}
```

Reply:
I could however this is affecting the coverage as it can't evaluate the content of `labels`. I can add it by skipping this section with `/* istanbul ignore next */` and allowing the arrow function with `tslint:disable-line jsx-no-lambda`.

Reply:
This should be ok.